addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java

```diff
@@ -40,7 +40,7 @@ protected Integer toInteger(Object object) {
             return (Integer) object;
         }
         try {
-            return Double.valueOf(ObjectUtils.toString(object)).intValue();
+            return Long.valueOf(Double.valueOf(ObjectUtils.toString(object)).longValue()).intValue();
```

Comment:
This line became really bad imho, double can be used for the same value, I don't suppose any change is needed on this line.